feature_engine/outliers/base_outlier.py

```diff
@@ -210,15 +224,20 @@ def fit(self, X: pd.DataFrame, y: Optional[pd.Series] = None):
         self.right_tail_caps_ = {}
         self.left_tail_caps_ = {}
 
+        if self.fold == "auto":
+            self.fold_ = self._calculate_auto_fold()
```

Comment:
We need to add `fold_` to the attributes list in the docstrings. Could you do that?